frontend/src/configs/common.js

```diff
@@ -1,6 +1,8 @@
 const PLAUSIBLE_DATA_DOMAIN_STAGING =
   "cellxgene.staging.single-cell.czi.technology";
 
+const DOCS_HOME_PAGE = "https://docs.cellxgene.cziscience.com/";
```

Comment:
Since this `common.js` file is intended for configs only, maybe we can move this external link to `routes.ts`!